### PR TITLE
/upload returns a JSON response instead of only headers

### PR DESCRIPTION
**Pull request by jotare**

### CODE REVIEW

File: nucliadb_models/nucliadb_models/writer.py [hunk 1]
```diff
@@ -112,3 +112,9 @@ class ResourceDeleted(BaseModel):
 
 
 ComminResourcePayload = Union[CreateResourcePayload, UpdateResourcePayload]
+
+
+class ResourceFileUploaded(BaseModel):
+    seqid: Optional[int] = None
+    rid: Optional[str] = None
```
Comment (lferran) [inline nucliadb_models/nucliadb_models/writer.py:119]: Some questions that come to my mind:

- Should this be `uuid` instead, to be "coherent" with `ResourceCreated` model?
- Should it be stored in `nucliadb_models`. What policy are we following?

File: nucliadb_writer/nucliadb_writer/api/v1/upload.py [hunk 2]
```diff
@@ -537,13 +538,15 @@ async def generate_buffer(storage_manager, request):
     except LimitsExceededError as exc:
         raise HTTPException(status_code=402, detail=str(exc))
 
-    headers = {}
-    headers["NDB-Seq"] = f"{seqid}"
-    headers["NDB-Resource"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}"
-    headers["NDB-Field"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}/field/{valid_field}"
+    response.headers["NDB-Seq"] = f"{seqid}"
+    response.headers["NDB-Resource"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}"
```
Comment (lferran) [inline nucliadb_writer/nucliadb_writer/api/v1/upload.py:542]: What about changing `resources` for the `RESOURCE_PREFIX` constant instead?

**Review by lferran — APPROVED** [review]
Some NIT comments, but LGTM! 👍🏼

File: nucliadb_writer/nucliadb_writer/api/v1/upload.py (hunk 2 above)
Comment (jotare) [reply nucliadb_writer/nucliadb_writer/api/v1/upload.py:542]: This headers will be removed, I'm uploading a new version

File: nucliadb_models/nucliadb_models/writer.py (hunk 1 above)
Comment (jotare) [reply nucliadb_models/nucliadb_models/writer.py]: Renamed `rid` to `uuid`

**Pull request merged**